src/core/server/plugins/types.ts

```diff
@@ -153,6 +153,12 @@ export interface PluginManifest {
    * Specifies whether plugin includes some server-side specific functionality.
    */
   readonly server: boolean;
+
+  /**
+   * Specifies directory names that can be imported by other plugins sharing
+   * the same instance of the @kbn/optimizer.
+   */
```

Comment:
could you add that is a temporary solution? we could even mark it as `@deprecated`